### PR TITLE
Improve chat page and fix mobile Spongie mark

### PR DESCRIPTION
**Pull request by stephan-gh**
Removed the background and replaced it with a full screen chat window that should be nicer to use.

There is a weird bug that is happening only **sometimes** on Firefox: It will show two scroll bars, one for the footer (intended) and one for the iframe content, even though it fully fits in the iframe. There seems to be no way to remove it and it doesn't always happen.

I haven't found a way to reproduce it reliably and it works as expected on all other browsers I've tested.
### Demo

A demo page is not available right now because it's taken by the downloads page - I have a few screenshots though:

screen shot 2016-10-17 at 20 11 38

More screenshots:
- [Start page](https://cloud.githubusercontent.com/assets/3035868/19450051/39018110-94a8-11e6-99be-9d9362f48cac.png)
- [Scrolling down to see the footer](https://cloud.githubusercontent.com/assets/3035868/19450091/4df46038-94a8-11e6-981d-e8e6598cf04d.png)

### CODE REVIEW

**Comment by progwml6** [comment]
@Minecrell can you post screenshots for the mobile issue you were describing?

File: public/assets/scss/_chat.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+.chat-page {
```
Comment (jamierocks) [inline public/assets/scss/_chat.scss:24]: `.chat-page footer {`

**Comment by stephan-gh** [comment]
@progwml6 When I fixed the chat page for #106 I didn't look at the HTML carefully enough so the `logo` class applied only for the Spongie mark on the chat page, not on the main homepage (because they have slightly different HTML structure).

On desktops that doesn't make a big difference because Spongie has the normal size there, however on mobile devices Spongie grew up to full width so it looked quite weird. I've fixed it by reverting my fix for the chat page (because I've completely removed the Spongie mark from there now).

Screenshot

File: public/assets/scss/_chat.scss (hunk 1 above)
Comment (stephan-gh) [reply public/assets/scss/_chat.scss:24]: Doesn't make a difference for the generated output. It looks nicer imo and makes it easier to add more things in the future.

**Pull request merged**